distributed/tests/test_client.py

```diff
@@ -7345,6 +7334,31 @@ async def test_dump_cluster_state_json(c, s, a, b, tmp_path, local):
         await c.dump_cluster_state(filename, format="json")
 
 
+@pytest.mark.parametrize("local", [True, False])
+@pytest.mark.parametrize("_format", ["msgpack", "yaml"])
+@pytest.mark.parametrize("workers", [True, False])
+@gen_cluster(client=True)
+async def test_inspect_cluster_dump(c, s, a, b, tmp_path, _format, local, workers):
+    filename = tmp_path / "foo"
+    if not local:
+        pytest.importorskip("fsspec")
+        # Make it look like an fsspec path
+        filename = f"file://{filename}"
+
+    A = da.ones(100, chunks=25)
```

Comment:
afaik, `dask.array` is an optional dependency. We have a bunch of tests that use `importorskip`. I would suggest to use plain futures, e.g.

```python
futs = c.map(inc, range(100))
await c.gather(futs)
```

for "practice" I would also suggest to construct a more interesting state, e.g. ensure there is a task in processing by using an event (below is pseudo / particularly awaits are missing)

```python
event = distributed.Event()

def blocked_inc(x, event):
    event.wait()
    return x + 1

inc_blocked = c.submit(blocked_inc, event=event)
cluster_dump()

assert inc_blocked.key in inspector.tasks_in_state("processing")
```

Reply:
changed